.github/ISSUE_TEMPLATE/user-story-template.md

```diff
@@ -25,6 +25,27 @@ Then
 - [ ] Checklist item
 - [ ] Meets the DOD
 
+**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)
+- [ ] User story is included
+- [ ] User role and type are identified
+- [ ] Acceptance criteria are included
+- [ ] Wireframes are included (if required)
+- [ ] Does this User Story need stakeholder approval?
+- [ ] Design / Solution is accepted by Product Owner
+- [ ] Dependencies are identified (technical, business, regulatory/policy)
+- [ ] Story has been estimated (under 13 spts)
+
+·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)
+- [ ] Passes developer unit and automated tests
+- [ ] UI meets accessibility requirements
+- [ ] Configuration changes are documented, documentation and designs are updated
+- [ ] Feature flagged if required
```

Comment:
What does this mean? Putting the feature behind a feature flag? We've never done this and it should be an exception, not the norm.

Reply:
Will remove the bullet point "Feature flagged if required".